### PR TITLE
removing whitelisted links from linkchecker

### PR DESCRIPTION
**Pull request by ayaleloehr**

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -184,22 +184,15 @@ if (options.watch) {
     allowRedirects: true,  // Don't require trailing slash for index.html links.
     warn: process.env.NODE_ENV !== 'production',
     allowRegex: new RegExp(
-        [ '/disability-benefits/',
-          '/disability-benefits/apply-for-benefits/',
-          '/disability-benefits/learn/',
-          '/disability-benefits/learn/eligibility/.*',
+        [ '/disability-benefits/apply-for-benefits',
```
Comment (ayaleloehr) [inline script/build.js]: I don't understand why this one is needed, but when I remove it I get:
```
Warning: Link is broken: href: "/disability-benefits/apply-for-benefits/active-duty/", text: "
              How do I file while on active duty?
            ", in file: disability-benefits/apply-for-benefits/help/index.html
```
Any thoughts @awong-dev or @bshyong ?

Comment (awong-dev) [inline script/build.js]: It's cause we have an actual broken link here.... :)

Comment (awong-dev) [inline script/build.js]: @crwallace this is what gets to be removed if we fix that link.

**Comment by emilyville** [comment]
Thinking this needs to be closed or [something else] so transferring ownership from albert to alex!

**Comment by ayaleloehr** [comment]
OBE. Thanks for flagging. Closing.

**Pull request closed without merging**